mitodl/micromasters
# Categorized FAQs

#### What are the relevant tickets?
#1190

#### What's this PR do?
Allow FAQ questions to be categorized

#### Where should the reviewer start?
Go into the CMS, click on the "Snippets" section, and make a FAQ category or two. Then add a few FAQs to a course, and assign them to categories

#### How should this be manually tested?
View the FAQ tab on the program page. Be sure to test with several different variations of questions and categories. Note that @roberthouse54 has asked me to just put the category in a `<h2>` tag and leave the styling to him, so the styling isn't very nice right now -- he will take care of that later (or perhaps push a commit to this pull request).

#### Screenshots (if appropriate)
<img width="383" alt="snippets section" src="https://cloud.githubusercontent.com/assets/132355/19095947/cad7f674-8a4d-11e6-8847-1514799bcf52.png">
<img width="896" alt="snippet FAQ categories" src="https://cloud.githubusercontent.com/assets/132355/19095949/cff3095a-8a4d-11e6-91c1-7de3205be86d.png">
<img width="522" alt="added a General FAQ category" src="https://cloud.githubusercontent.com/assets/132355/19095954/d53b887e-8a4d-11e6-8fdb-f510b0f9701c.png">
<img width="582" alt="selecting a category for a FAQ" src="https://cloud.githubusercontent.com/assets/132355/19095955/d8a92ae8-8a4d-11e6-8b4f-44affff1e082.png">
![setting up FAQs in a program](https://cloud.githubusercontent.com/assets/132355/19095959/df18bdbc-8a4d-11e6-96c2-196d50894c06.png)
![viewing FAQs in a program](https://cloud.githubusercontent.com/assets/132355/19095960/e080b15a-8a4d-11e6-93af-2eaf0a3d2882.png)
